src/ontodia/diagram/view.ts

```diff
@@ -437,6 +443,21 @@ export class DiagramView {
         this.listener.stopListening();
         this.disposed = true;
     }
+
+    loadEmbeddedElements = (id: string, iri: string): Promise<Dictionary<ElementModel>> => {
```

Comment:
I think `loadEmbeddedElements` should be part of `DiagramModel`, not `DiagramView`.

Also, the `id` parameter is unused in this method.